department-of-veterans-affairs/abd-vro
# introduce env variables for slack notifications

## What was the problem?

1. The app could not find the Slack notifications secrets in the environment variables
2. "automatedClaim" was waiting for the duration of the delay before returning. It should return immediately

Associated tickets or Slack threads:

- No associated ticket

## How does this fix it?[^1]

1. Infuse docker with environment variables
2. Return immediately after a call

## How to test this PR
NA


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
